third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/DotOpToLLVM/MMAv2.cpp

```diff
@@ -61,10 +61,28 @@ Value loadC(Value tensor, Value llTensor,
   return llTensor;
 }
 
+// The number of i32 registers owned by each thread along m, n, k dimensions.
+// For example, for m16n8k32 with i8 inputs, a thread owns 2, 1, and 2 registers
+// along m, n, k respectively.
+struct NumRegisters {
```

Comment:
passby nit: looks like this file is missing an anonymous namespace, would be nice to add it to not pollute the global namespace.

Reply:
Thanks! I'll put anonymous namespace in this file.